cluster/gce/config-test.sh

```diff
@@ -49,7 +49,7 @@ CLUSTER_NAME="${CLUSTER_NAME:-${INSTANCE_PREFIX}}"
 MASTER_NAME="${INSTANCE_PREFIX}-master"
 MASTER_TAG="${INSTANCE_PREFIX}-master"
 NODE_TAG="${INSTANCE_PREFIX}-minion"
-CLUSTER_IP_RANGE="${CLUSTER_IP_RANGE:-10.245.0.0/16}"
+CLUSTER_IP_RANGE="${CLUSTER_IP_RANGE:-10.245.0.0/14}"
```

Comment:
This is the same as `10.244.0.0/14`. If you want test to remain a distinct range from regular clusters we should change this to `10.248.0.0/14`.